packages/react-charting/src/components/TreeChart/TreeChart.base.tsx

```diff
@@ -169,9 +169,12 @@ class StandardTree {
     gap: number,
   ): string {
     // gap adds ratio for parent.y to child.y
-    const path = `M${childX + rectWidth / 2},${childY - gap} H${parentX + rectWidth / 2} V${
-      parentY + rectHeight + gap / 2
-    }`;
+    const path =
+      parentX > childX
+        ? `M${childX},${childY - gap} H${parentX + rectWidth / 2}
+    V${parentY + rectHeight + gap / 2}`
+        : `M${childX + rectWidth},${childY - gap} H${parentX + rectWidth / 2}
```

Comment:
Avoid code duplication. Change the value used as rectWidth and use a single path definition

Reply:
This is completed.